src/Redux/Slices/DiscoveryDroppables.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { appliedItemDrag, getDroppablesGridState } from 'Utils/DiscoveryUtils/droppablesStateUtils';
+import gridConfigs from 'Utils/DiscoveryUtils/gridsConfigs';
+
+const initialState = { hasPlayerDragged: false };
+
+const DiscoveryDroppablesSlice = createSlice({
+	name: 'DiscoveryDroppables',
+	initialState,
+	reducers: {
+		initializeState: (state, action) => {
+			let newState = { hasPlayerDragged: false };
```

Comment:
please avoid duplicate code by making initiastate a function that generaes this object